apache/superset
# fix(dockerfile): podman compose compatibility with BUILDPLATFORM

### SUMMARY

Fixes #31488.

Using a different variable, `TARGET_BUILDPLATFORM`, inside the `Dockerfile` to avoid name conflict/redefining `BUILDPLATFORM` which is an issue in podman.
```Dockerfile
ARG TARGET_BUILDPLATFORM=${BUILDPLATFORM:-amd64}
```

### TESTING INSTRUCTIONS

Run the command
```
podman compose -f docker-compose-non-dev.yml up --build
```
and observe that the error
> Error: reading multiple stages: attempted to redefine "BUILDPLATFORM": invalid argument

Is no longer there.

### ADDITIONAL INFORMATION
